docs/_docs/analysis.md

```diff
@@ -351,6 +351,45 @@ Calculates average molecular multipolar moments and their fluctuations.
 -------------- | ----------------------
 `nstep`        | Interval between samples.
 
+The output from the multipole analysis gives the following:
+`multipole`    | Description
+-------------- | ----------------------
+` C    `       | Capacitance  
+ `Z    `       | Charge/Valency 
+ `Z2    `      | Squared charge/valency
+ `μ     `      | Dipole moment 
+ `μ2    `      | Squared dipole moment
+
+ The capacitance, C, is defined accordingly: 
```

Comment:
"C" --> "_C_" to make it italic. Same for the other cases.

Reply:
Perhaps also add units in the table, i.e. "e", "eÅ"

Reply:
Unsure about unit for capacitance, is eV⁻¹ correct?